tests/Api/Utils/OrderPlacerTrait.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Utils;
+
+use Sylius\Bundle\ApiBundle\Command\Cart\AddItemToCart;
+use Sylius\Bundle\ApiBundle\Command\Cart\PickupCart;
+use Sylius\Bundle\ApiBundle\Command\Checkout\AddressOrder;
+use Sylius\Bundle\ApiBundle\Command\Checkout\ChoosePaymentMethod;
+use Sylius\Bundle\ApiBundle\Command\Checkout\ChooseShippingMethod;
+use Sylius\Bundle\ApiBundle\Command\Checkout\CompleteOrder;
+use Sylius\Component\Core\Model\Address;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+trait OrderPlacerTrait
+{
+    protected function placeOrder(string $tokenValue, array $header): void
+    {
+        /** @var MessageBusInterface $commandBus */
+        $commandBus = $this->get('sylius.command_bus');
+
+        $pickupCartCommand = new PickupCart($tokenValue, 'en_US');
+        $pickupCartCommand->setChannelCode('WEB');
+        $commandBus->dispatch($pickupCartCommand);
+
+        $addItemToCartCommand = new AddItemToCart('MUG_BLUE', 3);
+        $addItemToCartCommand->setOrderTokenValue($tokenValue);
+        $commandBus->dispatch($addItemToCartCommand);
+
+        $address = new Address();
+        $address->setFirstName('John');
+        $address->setLastName('Doe');
+        $address->setCity('New York');
+        $address->setStreet('Avenue');
+        $address->setCountryCode('US');
+        $address->setPostcode('90000');
+
+        $addressOrderCommand = new AddressOrder('sylius@example.com', $address);
+        $addressOrderCommand->setOrderTokenValue($tokenValue);
+        $commandBus->dispatch($addressOrderCommand);
+
+        $this->client->request('GET', '/api/v2/admin/orders/nAWw2jewpA', [], [], $header);
```

Comment:
I'm wondering... This method is a setup for the test, we're using commands directly to create an order and complete it. Should we use an API there? If something is broken in the used endpoint, the setup would not work as well (which it shouldn't). Can't we just get an order from the repository, to make this setup method interface agnostic?